pkgs/applications/science/math/mxnet/default.nix

```diff
@@ -2,6 +2,7 @@
 , opencv3, gtest, blas, gomp, llvmPackages, perl
 , cudaSupport ? config.cudaSupport or false, cudatoolkit, nvidia_x11
 , cudnnSupport ? cudaSupport, cudnn
+, cudaCapabilities ? [ "3.7" "5.0" "6.0" "7.0" "7.5" "8.0" "8.6" ]
```

Comment:
This is so as to remove `sm_30` from mxnet's default gencode list. The `sm_30` has been deprecated in cuda 11. I'm not inventing any naming schemes here and just using the format that the package accepts - we'll need to rework this when we introduce `config.cudaCapabilities` or anything similar